src/commands/Minion/nightmare.ts

```diff
@@ -193,27 +279,34 @@ export default class extends BotCommand {
 			Time.Minute * 5,
 			Time.Minute * 30
 		);
-		this.checkReqs(users, NightmareMonster, quantity);
+		this.checkReqs(users, NightmareMonster, quantity, isPhosani);
 
 		duration = quantity * perKillTime - NightmareMonster.respawnTime!;
 
 		const totalCost = new Bank();
-		if (NightmareMonster.healAmountNeeded) {
-			for (const user of users) {
-				const [healAmountNeeded] = calculateMonsterFood(NightmareMonster, user);
-				const { foodRemoved } = await removeFoodFromUser({
-					client: this.client,
-					user,
-					totalHealingNeeded: Math.ceil(healAmountNeeded / users.length) * quantity,
-					healPerAction: Math.ceil(healAmountNeeded / quantity),
-					activityName: NightmareMonster.name,
-					attackStylesUsed: ['melee']
-				});
-				totalCost.add(foodRemoved);
+		let soloFoodUsage: Bank | null = null;
+		for (const user of users) {
+			const [healAmountNeeded] = calculateMonsterFood(NightmareMonster, user);
+			const cost = this.perUserCost(user, isPhosani, quantity);
+			if (!user.owns(cost)) {
+				return msg.channel.send(`${user} doesn't own ${cost}.`);
 			}
+			let healingMod = isPhosani ? 1.5 : 1;
```

Comment:
Sang staff should give a slight healing buff, maybe 1.4 instead of 1.5. (it should be a bigger buff for regular nightmare)

(Phosani only:) Also, with high KC, you should use less food, because most damage can be mitigated, only the final phase has unavoidable damage.  (Husks can be prayed against, but are not the priority, so you can take a small amount of damage from them, also.) That said, mistakes are common, especially with less experience. 

[Normal Nightmare prayers only block a % of  the nightmares damage, while Phosani, they block 100% because there's so many more of them]